packages/riipen-ui/src/components/editor/ControlButton.jsx

```diff
@@ -0,0 +1,97 @@
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import React from "react";
+
+import ThemeContext from "../../styles/ThemeContext";
+import withClasses from "../../utils/withClasses";
+
+class ControlButton extends React.Component {
+  static propTypes = {
+    /**
+     * If the control is active.
+     *
+     */
+    active: PropTypes.bool.isRequired,
+
+    /**
+     * The display for the button. We use a string or <Icon />.
+     *
+     */
+    label: PropTypes.node.isRequired,
+
+    /**
+     * Function to execute on control button click.
+     *
+     */
+    onClick: PropTypes.func.isRequired,
+
+    /**
+     * The width of the control button.
+     *
+     */
+    width: PropTypes.oneOf(["small", "medium", "large"])
```

Comment:
I think we should change this prop name to `size` instead of `width`. It would be more inline with our naming conventions on other components that have small / medium / large options.